src/hooks/useLists.ts

```diff
@@ -8,11 +8,11 @@ import { PULLING_STATUS_DELAY } from './useRefresh/constants';
 let pageCount = 0;
 let totalRecordCount = 0;
 
-export const useLists = (filters: Array<string>, idsToTrack?: Array<string>, size?: number, offset?: number) => {
+export const useLists = (filters: Array<string>, idsToTrack?: Array<string>, size?: number, offset?: number, updatedAsOf?: string) => {
```

Comment:
What does it means updatedAsOf?

Reply:
It's the same name as the API parameter. Agree it's not the best name, but since this hook directly interacts with the API, I'd like to keep it.

Reply:
Is parameter consumes functions on FE side so we easily rewrite the name here to the better one. I won't affect BE contracts

Reply:
We can rewrite the name. I figured it would be easier to use the same name so we know which value is mapped to the API.

Reply:
Renamed as `listsLastFetchedTimestamp`